libraries/cms/application/site.php

```diff
@@ -651,6 +621,25 @@ protected function initialiseApp($options = array())
 	}
 
 	/**
+	 * Language check
+	 *
+	 * @param  string  Language to check.
+	 *
+	 * @return  mixed  string  The passed language if it is an available language.
+	 *                 null    If the language is not an available language.
+	 *
+	 * @since   3.4.2
+	 */
+	protected function checkLang($lang)
```

Comment:
Thanks @Bakual!
A private method is more than enough in this case, but just to be prepared for the future,
would be OK to **add** a public method to a class (without modifying other methods behavior)?